jenkins/catapult/jenkins/catapult-client-build-catapult-project-windows.groovy

```diff
@@ -0,0 +1,241 @@
+pipeline {
+	agent {
+		label 'windows-xlarge-agent'
+	}
+
+	parameters {
+		gitParameter branchFilter: 'origin/(.*)', defaultValue: "${env.GIT_BRANCH}", name: 'MANUAL_GIT_BRANCH', type: 'PT_BRANCH'
+		choice name: 'COMPILER_CONFIGURATION',
+			choices: ['msvc-2022', 'msvc-2019'],
+			description: 'compiler configuration'
+		choice name: 'BUILD_CONFIGURATION',
+			choices: ['tests-conan'],
+			description: 'build configuration'
+		choice name: 'OPERATING_SYSTEM',
+			choices: ['windows'],
+			description: 'operating system'
+
+		string name: 'TEST_IMAGE_LABEL', description: 'docker test image label', defaultValue: ''
+		choice name: 'TEST_MODE',
+			choices: ['test', 'none'],
+			description: 'test mode'
+		choice name: 'TEST_VERBOSITY',
+			choices: ['suite', 'test', 'max'],
+			description: 'output verbosity level'
+
+		booleanParam name: 'SHOULD_PUBLISH_BUILD_IMAGE', description: 'true to publish build image', defaultValue: false
+	}
+
+	environment {
+		DOCKER_URL = 'https://registry.hub.docker.com'
+		DOCKER_CREDENTIALS_ID = 'docker-hub-token-symbolserverbot'
+	}
+
+	options {
+		ansiColor('css')
+		timestamps()
+	}
+
+	stages {
+		stage('prepare') {
+			stages {
+				stage('prepare variables') {
+					steps {
+						script {
+							buildImageLabel = '' != TEST_IMAGE_LABEL ? TEST_IMAGE_LABEL : getBuildImageLabel()
+							buildImageFullName = "symbolplatform/symbol-server-test:${buildImageLabel}"
```

Comment:
notice that you're using lowerCamelCase for vars where most existing jobs are using snake_case. might want to cleanup the older jobs to use lowerCamelCase too as post merge task.